src/reward/reward.service.ts

```diff
@@ -18,4 +18,17 @@ export class RewardService {
       return { error };
     }
   }
+
+  getRewardBindingByAccountId(accountId){
+    try {
+      return this.rewardRepository.findOne({
+        where: { 
+          reward_type : 'Registered User',
+          address: accountId
+        }
+      })
+    } catch (error) {
+      return error
```

Comment:
Don't return the error, throw a new one. Define the expected return response of the function.